core/base/auction/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 ttk_add_base_library(auction
 
-  SOURCES Auction.cpp 
+  SOURCES Auction.cpp
   HEADERS Auction.h AuctionImpl.h AuctionActor.h
-	LINK triangulation persistenceDiagram kdTree)
```

Comment:
have you changed all LINK keywords by DEPENDS keywords systematically?

Reply:
Yes, in order to stay consistent with the wrappers so we do not have to use LINK here and DEPENDS on the vtk wrapper for the same purpose.